flexible_event_config.md

```diff
@@ -76,32 +79,8 @@ In addition to the parameters that were added in Phase 1, we will add one additi
       "end_times": [<int>, ...],
     }
 
-    // Represents an operator that summarizes the triggers attributed to the source.
-    // count: number of triggers attributed
-    // value_sum: sum of the value of triggers
-    // Defaults to "count"
-    "summary_operator": <one of "count" or "value_sum">,
-
-    // Represents a bucketization of the integers from [0, MAX_UINT32], encoded as
-    // a list of integers where new buckets begin (excluding 0 which is
-    // implicitly included).
-    // It must consist of strictly increasing positive integers, and its length
-    // must be <= max_event_level_reports.
-    //
-    // e.g. [5, 10, 100] encodes the following ranges:
-    // [[0, 4], [5, 9], [10, 99], [100, MAX_UINT32]]
-    //
-    // At the end of each reporting window, triggers will be summarized into an
-    // integer which slots into one of these ranges. Reports will be sent for
-    // every new range boundary that is crossed. Reports will never be sent for
-    // the range that includes 0, as every source is initialized in this range.
-    //
-    // If omitted, then represents a trivial mapping
-    // [1, 2, ... , max_event_level_reports].
-    "summary_buckets": [<bucket start>, ...]
-  }, {
-    // Next trigger_spec
-  }, ...],
+   }
```

Comment:
The `trigger_specs` field is not part of the "Phase 1" and "Phase 2", as written in this document. That field will only be added along with `summary_buckets` and per-trigger-data report windows.

Reply:
Hi, is there any estimate when this feature will be available?

Reply:
@apasel422 I have updated the changes so that `trigger_specs` is no longer included as part of phase 1 and phase 2

@evazkj there is currently no timeline for the "ideas for future iteration", but we are open to additional feedback from the ecosystem regarding these, and will update when we do have a more concrete timeline.